### ISSUE TITLE
Dependency on Readonly or on Readonly::Array?

### ISSUE THREAD

**Issue by andk** [toddr/IPC-Run#127]
Executive summary: Please drop the dependencies on Readonly::Array and Readonly.

TLDR:

Grepping through all *.pm files does not confirm that there is a dependency.

Readonly is used in t/readonly.t but the test gets skipped if the 'use Readonly' does not succeed.

Readonly::Array is a package within the file Readonly.pm, so it does not ever exist in a file Readonly/Array.pm.

Readonly::Array is indexed on the CPAN but older vorsions of the CPAN client complain that Readonly::Array could not be loaded.

Bottom line: a not-dependency forces all users to install Readonly without need. Additionally, older software may have issues with the dependency Readonly::Array.

**Comment by mohawk2** [comment]
Sounds like `Readonly::Array` should be dropped, and `Readonly` switched to a `recommends`?
